### PR TITLE
Enable basic PHPSTAN checks

### PR DESCRIPTION
**Pull request by dennisameling**
Sets up basic PHPSTAN checks. Setting the level higher than 1 causes quite some errors/warnings, so I'm leaving it at 1 for now.

### CODE REVIEW

**Comment by RCheesley** [comment]
@dennisameling do you need me to test this or is it just a case of getting the green tick to merge? 🤓 ✅

File: tests/Api/CampaignsTest.php [hunk 1]
```diff
@@ -471,7 +471,7 @@ public function testCampaignContactEditEvent()
                 $date = new \DateTime($log['triggerDate'], new \DateTimeZone('UTC'));
                 $this->assertEquals($log['triggerDate'], $date->format('c'));
             } else {
-                $this->assertFalse(false, 'Event ID not recognized in the log.', var_export($event, true));
+                $this->assertFalse(var_export($event, true), 'Event ID not recognized in the log.');
```
Comment (escopecz) [inline tests/Api/CampaignsTest.php:474]: This doesn't make much sense. The previous version did not make sense either. I expect the intent was to fail the test if this assert will run. How about this:
```suggestion
                $this->fail('Event ID not recognized in the log.'.var_export($event, true));
```

**Review by escopecz — CHANGES_REQUESTED** [review]
Found 1 line that doesn't/didn't make sense and suggested change. Let me know if it make sense. Otherwise this is good to go 👍

File: tests/Api/CampaignsTest.php (hunk 1 above)
Comment (escopecz) [reply tests/Api/CampaignsTest.php:474]: @dennisameling WDYT?

**Comment by RCheesley** [comment]
@escopecz some conflicts with this PR - suggest that we have someone from @mautic/core-team pick this up if we want to merge it?

**Pull request closed without merging**